RHEL/7/templates/static/ansible/clean_components_post_updating.yml

```diff
@@ -0,0 +1,10 @@
+# platform = Red Hat Enterprise Linux 7
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+
+- lineinfile:
+    path: /etc/yum.conf
+    regexp: '^clean_requirements_on_remove'
```

Comment:
Please include '=' in the regexp to prevent `clean_requirement_on_remove_special_case` being changed.